README.rst

```diff
@@ -1,29 +1,60 @@
-Linear Stage Component
-++++++++++++++++++++++
+##############
+ts-LinearStage
+##############
 
-This is the readme for the linear stage python api.
+The LinearStage is a CSC for the Vera C. Rubin Observatory.
+It controls a Zaber linear motor.
 
-Manual
-------
-This is the manual_ for the linear stage from Zaber_.
+Installation
+============
 
-.. _manual: https://www.zaber.com/manuals/A-LST
-.. _Zaber: https://www.zaber.com/
+.. code::
 
-Installation
+    setup -kr .
+    scons
+
+.. code::
+
+    pip install .[dev]
+    pytest --cov lsst.ts.LinearStage -ra
+
+Requirements
 ------------
-This package should be installed using the eups method.
+Run the ``develop-env`` docker image.
+
+Usage
+=====
+
+.. code::
+
+    from lsst.ts import salobj
+    linear_stage = salobj.Remote(name="LinearStage", domain=salobj.Domain(), index=1)
```

Comment:
I think explanation of the indices would be good? How to use two axes at once etc.

Reply:
Do you mean two stages?

Reply:
Yes. Essentially using both simultaneously.